index.js

```diff
@@ -220,6 +220,27 @@ module.exports = {
 				.catch(err => console.error(err));
 		},
 
+		list({ max = 100, fromAddress = undefined, network = 'mainnet' } = {}) {
```

Comment:
Do we really need this? It can be achieved with `exchanges.since`, providing `fromAddress` and `minTimestamp: 0`

Reply:
Yeah ok good call. I don't know why I though `exchanges.since` was only time denominated